templates/thank-you.html

```diff
@@ -3,42 +3,38 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1zAmRxMVFQhUPv9jBjVHJPnWfJMjiB78dQfY__h4pE7o/edit{% endblock meta_copydoc %}
 
 {% block content %}
-<main id="main-content">
 
-  <section class="p-strip is-deep">
-    <div class="row u-equal-height">
-      <div class="col-7">
-        <h1>Thank you for contacting us about MicroK8s</h1>
-        <p class="p-heading--4">A member of our team will be in touch within one working day.</p>
-      </div>
-      <div class="col-5 u-align--center u-vertically-center u-hide--small u-hide--medium">
-        <img src="https://assets.ubuntu.com/v1/52d53696-picto-thankyou-midaubergine.svg" alt="Smile" width="200" height="200">
-      </div>
+<section class="p-strip is-deep">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h1>Thank you for contacting us about MicroK8s</h1>
+      <p class="p-heading--4">A member of our team will be in touch within one working day.</p>
     </div>
-  </section>
-
-  <section class="p-strip--light">
-    <div class="row p-divider">
-      <div class="col-4 p-divider__block">
-        <h3 class="p-heading--4">Ubuntu Advantage</h3>
-        <p>Purchase our desktop support and access Ubuntu experts whenever you need.</p>
-        <p><a  href="https://buy.ubuntu.com">Buy Ubuntu Advantage</a></p>
-      </div>
-
-      <div class="col-4 p-divider__block">
-        <h3 class="p-heading--4">Ubuntu OpenStack</h3>
-        <p>Learn about Canonical&rsquo;s industry leading, private cloud infrastructure platform based on Openstack.</p>
-        <p><a href="/openstack">Learn about Ubuntu OpenStack&nbsp;&rsaquo;</a></p>
-      </div>
+    <div class="col-5 u-align--center u-vertically-center u-hide--small u-hide--medium">
+      <img src="https://assets.ubuntu.com/v1/52d53696-picto-thankyou-midaubergine.svg" alt="Smile" width="200" height="200">
```

Comment:
```suggestion
      <img src="https://assets.ubuntu.com/v1/52d53696-picto-thankyou-midaubergine.svg" width="200" height="200">
```
As this is decorative, we could omit the alt text entirely https://accessibility.huit.harvard.edu/describe-content-images

Reply:
For sure, that alt text actually existed in the page beforehand. The only change I made here was to remove the double main element